tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "declaration": true
   },
   "include": ["src/**/*.ts"],
-  "exclude": ["node_modules", "test/**/*.ts"]
```

Comment:
exclude here means `@types` aren't used, so things like `set-cookie-parser` don't get the correct types in local development

Reply:
Not sure what you mean by that. `exclude` means that this config won't apply to `*.ts`/`*.js` files inside the `node_modules` directory. Our source modules listed in `include` will _still_ have the types from `node_modules/@types` applied, incl. during the compilation. 

Are you encountering a particular issue here?

Reply:
I'll double check later today.  typechecking was failing in my editor for `set-cookie-parser` which had types installed via `@types/set-cookie-parser` but maybe something else was going on there

Reply:
not sure what happened here, maybe just vscode got stuck, seems to work fine without this change now

Reply:
Glad to hear that got resolved! So no changes to `tsconfig.json` are needed, in the end, right?